### ISSUE TITLE
Missing support for PSR Log 2 and 3

### ISSUE THREAD

**Issue by Brenneisen** [Vonage/vonage-php-sdk-core#299]

**Comment by SecondeJK** [comment]
@Brenneisen This issue does not use the issue reporting template and contains little information. Could you please edit the issue with more details? Thanks

**Comment by BafS** [comment]
The psr/log requirement for this package is `"psr/log": "^1.1"`, it would be great to support version [2 and 3](https://github.com/php-fig/log/tags).

**Comment by SecondeJK** [comment]
Right, I'll inject some newer loggers in to make sure they work OK, make any changes required and bump the versions on this then, but no ballpark estimate when it'll be done.

**Comment by SecondeJK** [comment]
Merged, will be released as part of 3.0.0
